src/pages/ErrorPage.jsx

```diff
@@ -0,0 +1,14 @@
+import { HomeButton } from "./HomeButton"
+import { Link } from "react-router-dom"
+import "./errorpage.css" 
+
+export const ErrorPage = () => {
+  return (
+    <div className="error-page">
+    <p className="error-text">Oops! Something went wrong!</p>
```

Comment:
Maybe this would be solved by adding a netlify.toml-file in your project? 
And in this you could write: 

```
[[redirects]]
 from = "/*"
 to = "/index.html"
 status = 200
```

Not sure what this does, but Diego talked about it in his Q&A... it redirects the user from pages that don't exist in some way?